templates/cisco_ios_show_interfaces_status.textfsm

```diff
@@ -1,17 +1,26 @@
 Value PORT (\S+)
 Value NAME (.+?)
-Value STATUS (err-disabled|disabled|connected|notconnect|inactive)
+Value STATUS (err-disabled|disabled|connected|notconnect|inactive|up|down)
 Value VLAN (\S+)
 Value DUPLEX (\S+)
 Value SPEED (\S+)
 Value TYPE (.*)
+Value MODE (\S+)
```

Comment:
Why was `MODE` added? I don't see it in the header of the output.

Reply:
Renamed to FC_MODE  it indicates the MODE of the Fiber channel e.g. "FP" , not the same as type, as TYPE refers to the physical port type.

Reply:
To show the FC port mode config,  renamed to FC_MODE.